docs/content/v2024.2/deploy/checklist.md

```diff
@@ -194,51 +193,50 @@ Operating a distributed database on ephemeral storage requires careful planning.
+#### Universe design and configuration best practices
 
 To safely and effectively use ephemeral disks with YugabyteDB, follow these design and configuration guidelines:
 
-- Use sufficient replication and redundancy: Always use a minimum of Replication Factor (RF) of 3 for any production universe (and for all tablespaces for which you might override the RF) when using ephemeral disks. Due to the increased risk of "quorum-loss" scenarios, an even higher RF is recommended even higher RF if your environment and performance tolerances allow. For example, an RF5 deployment can survive node failures in two fault domains without data loss (at the cost of more storage and write overhead).
+- Have enough replication and redundancy: Always use a minimum replication factor (RF) of 3 for any production universe (and for all tablespaces for which you might override the RF) when using ephemeral disks. Due to the increased risk of "quorum-loss" scenarios, RF 5 or greater is recommended if your environment and performance tolerances allow. An RF 5 deployment can survive the failure of two fault domains without data loss (at the cost of more storage and write overhead).
 
-- Distribute across zones: Spread nodes across availability zones to prevent a single zone outage drop all replicas of a tablet, and therefore cause data loss. Spread your cluster across Availability Zones (AZs), and (if you are using tablespaces to override replication settings for particular tables), also spread each tablespace across multiple AZs.
+- Distribute across zones: Spread nodes across availability zones so that a single zone outage won't result in all replicas of a tablet being dropped, causing data loss. Likewise, if you are using tablespaces to override replication settings for particular tables, spread each tablespace across multiple zones.
 
-- Persistent boot disk for OS and binaries: Configure YugabyteDB nodes so that the YugabyteDB software, configuration files, and logs reside on a persistent disk, and only the data directory is on the ephemeral disk. This design allows for quicker provisioning of nodes back into the cluster. It's also recommended to place logs on an independent persistent disk that's not the boot disk. Keeping logs persistent retains debugging information in case of a failure, but placing them on an independent, non-boot disk avoids the possibility of filling up the boot disk.
+- Persistent boot disk for OS and binaries: Configure YugabyteDB nodes so that the YugabyteDB software, configuration files, and logs reside on a persistent disk, and only the data directory is on the ephemeral disk. This design allows for quicker provisioning of nodes back into the cluster. It's also recommended to place logs on an independent persistent disk that's not the boot disk. Keeping logs persistent retains debugging information in case of a failure, and placing them on an independent, non-boot disk avoids the possibility of filling up the boot disk.
```

Comment:
```suggestion
- Persistent boot disk for OS and binaries: Configure YugabyteDB nodes so that the YugabyteDB software, configuration files, and logs reside on a persistent disk, and only the data directory is on the ephemeral disk. This design allows for quicker provisioning of nodes back into the cluster. In addition, put logs on an independent persistent disk that's not the boot disk. Keeping logs persistent retains debugging information in case of a failure, and placing them on an independent, non-boot disk avoids the possibility of filling up the boot disk.
```